cpu/stm32/Kconfig

```diff
@@ -32,7 +32,12 @@ config HAS_BOOTLOADER_STM32
     help
         Indicates that the stm32 bootloader is being used.
 
+config ERROR_MODULES_CONFLICT
+    default "On STM32, the RTC and RTT map to the same hardware peripheral." if MODULE_PERIPH_RTC && MODULE_PERIPH_RTT
+    depends on CPU_STM32
+
```

Comment:
This addition seems wrong. It declares RTC and RTT as conflicting features for all STM32 cpus but that's not the case. Only STM32F1 is in that situation.

Is this a bug @leandrolanzieri ?

Reply:
I think it should be `depends on CPU_FAM_F1`

Reply:
Yes, you're right. It should only apply for F1. I'll change it.

Reply:
See #15834